Makefile

```diff
@@ -165,11 +169,23 @@ vagrantup: $(ALL) ## Run 'vagrant up' to (re-)provision a development environmen
 help: ## Show this help message
 	@echo "The following targets are available:"
 	@echo
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -Eh '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
 
+.PHONY: basecentos-vars
+basecentos-vars:
+	@if [[ `cat $(BASE_CENTOS_IMAGE) 2>&1` != $(BASE_CENTOS_IMAGE_NAME)$(BASE_CENTOS_IMAGE_SHA256) ]]; then \
+        echo -n $(BASE_CENTOS_IMAGE_NAME)$(BASE_CENTOS_IMAGE_SHA256) > $(BASE_CENTOS_IMAGE) ; \
+    fi
+
+$(BASE_CENTOS_IMAGE):
+	docker build \
+		-t base-centos-image \
+        --build-arg BUILD_IMAGE=$(BASE_CENTOS_IMAGE_NAME) \
+		--build-arg BUILD_IMAGE_SHA256=$(BASE_CENTOS_IMAGE_SHA256) \
+		images/base
 
-$(BUILD_ROOT)/package-build-container: packages/Dockerfile packages/entrypoint.sh
+$(BUILD_ROOT)/package-build-container: packages/Dockerfile packages/entrypoint.sh $(BASE_CENTOS_IMAGE) | basecentos-vars
```

Comment:
The usage of the "order only prerequisite" seems odd. You usually don't want to use a PHONY target for that (https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html)